docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:62e79e54d43aa1762ec68a6661872320a39517bd-hail-2ea2615a797a5aff72d20d9d12a2609342846a07
```

Comment:
Should we add a `latest` tag to the `driver` image, so we don't need to hardcode a version here (which will immediately become obsolete)?

Reply:
Yes, that would be amazing!

Reply:
https://github.com/populationgenomics/analysis-runner/pull/532